src/Lumi/Components2/Text.purs

```diff
@@ -29,7 +29,7 @@ import Lumi.Components.Color (ColorMap)
 import Lumi.Styles (Style, StyleModifier)
 import Lumi.Styles as S
 import Lumi.Styles.Theme (LumiTheme(..), TextMap, textFontSize, textLineHeight, textMargin, useTheme)
-import React.Basic (JSX, ReactComponent)
+import React.Basic.Classic (JSX, ReactComponent)
```

Comment:
Shouldn't we be importing from `React.Basic` here instead? Since this module uses the new hooks API.